src/Components/ActivitiesCarousel/ActivitiesCarousel.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Carousel, Image } from 'antd';
+import { LeftOutlined, RightOutlined, CameraOutlined } from '@ant-design/icons';
+import './ActivitiesCarousel.css';
+
+export default function ActivitiesCarousel({ actPicArray }) {
+  return (
+    <div className="carousel-act">
+      <Carousel
+        arrows
+        nextArrow={<RightOutlined />}
+        prevArrow={<LeftOutlined />}
+      >
+        {actPicArray.map((pic, index) => (
+          <div className="carousel-item" key={index}>
+            <Image width={400} src={pic.imgsrc} />
```

Comment:
you can use this size
```suggestion
            <Image width={'100%'} src={pic.imgsrc} />
```